### DISCUSSION TITLE
Does swup work with GitHub Pages

### DISCUSSION THREAD

**Discussion by DylanDevelops** [swup/swup#516] [Help] [answered]
I was curious if swup works with github pages coded in HTML and CSS. It says github only hosts static pages and I was having issues getting swup to work with my sight and I wanted to see if that had something to do with it.

**Comment by daun** [comment, marked as answer]
Yes, it should work just fine. Swup is made for static sites and GitHub pages doesn't have any restrictions on ajax requests, as far as I know.

**Reply by DylanDevelops** [reply]
Any idea why this isn't working? https://github.com/swup/swup/discussions/514#discussioncomment-3664691
